### ISSUE TITLE
Typo in python/dispatch.py

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#11130]
|<img alt="loic's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| loic reported|
|-|-|
|Trac ID|trac#1571|
|Type|defect|
|Created|2006-03-14 14:44:54Z|

http://twistedmatrix.com/trac/browser/trunk/twisted/python/dispatch.py?rev=16293

shows that DeprecationgWarning is used instead of DeprecationWarning.

<details><summary>Searchable metadata</summary>

```
trac-id__1571 1571
type__defect defect
reporter__loic loic
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1142347494000000 1142347494000000
changetime__1142353515000000 1142353515000000
version__2_0 2.0
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set status to `closed`|
|-|-|

(In [16294]) Spell DeprecationWarning correctly - Fixes [#11130](https://github.com/twisted/twisted/issues/11130)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
